src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/PseRespondToTribunalService.java

```diff
@@ -40,6 +45,39 @@ public class PseRespondToTribunalService {
     private static final String RULE92_ANSWERED_YES =
         "You have responded to the tribunal and copied your response to the other party.\r\n\r\n";
 
+    /**
+     * Create fields for application dropdown selector.
+     * @param caseData contains all the case data
+     */
+    public DynamicFixedListType populateSelectDropdown(CaseData caseData) {
+        if (CollectionUtils.isEmpty(caseData.getSendNotificationCollection())) {
+            return null;
+        }
+
+        return DynamicFixedListType.from(caseData.getSendNotificationCollection().stream()
+            .filter(r -> populateSelectDropdownFilterNotify(r)
+                && populateSelectDropdownFilterRespond(r))
+            .map(r ->
+                DynamicValueType.create(
+                    r.getValue().getNumber(),
+                    r.getValue().getNumber() + " " + r.getValue().getSendNotificationTitle()
+                )
+            )
+            .collect(Collectors.toList()));
+    }
+
+    private boolean populateSelectDropdownFilterNotify(SendNotificationTypeItem sendNotificationTypeItem) {
+        return RESPONDENT_ONLY.equals(sendNotificationTypeItem.getValue().getSendNotificationNotify())
+            || BOTH_PARTIES.equals(sendNotificationTypeItem.getValue().getSendNotificationNotify());
+    }
+
+    private boolean populateSelectDropdownFilterRespond(SendNotificationTypeItem sendNotificationTypeItem) {
+        return sendNotificationTypeItem.getValue().getRespondCollection() == null
```

Comment:
We can use this piece of code in https://github.com/hmcts/et-ccd-callbacks/pull/726/files TseHelper to avoid code duplication?

Reply:
Thank you for your comment.
Although they look similar, they are different classes from different epic.
It might be better to keep them within their epic.